src/ast.ml

```diff
@@ -1,10 +1,20 @@
+type attributes =
+  {
+    id: string option;
+    classes: string list;
+    attributes: (string * string) list;
+  }
+
 type 'a link_def =
   {
     label: 'a;
     destination: string;
     title: string option;
+    attributes: attributes;
   }
 
+let empty_attributes = {id=None; classes=[]; attributes=[]}
```

Comment:
let's make an `Attributes` module with a type `t` and a value `empty`

Reply:
I agree, should have thought about it before after doing all those modules :)